common/scala/src/main/scala/whisk/core/containerpool/logging/DockerToActivationLogStore.scala

```diff
@@ -29,6 +28,7 @@ import whisk.common.TransactionId
 import whisk.core.containerpool.Container
 import whisk.core.entity.{ActivationLogs, ExecutableWhiskAction, Identity, WhiskActivation}
 import whisk.http.Messages
+import whisk.core.database.UserContext
```

Comment:
is this class tightly coupled with the `database` package?  on the surface, it seems like it could go into some place else.

Reply:
@mdeuser, the ActivationStore was moved to the database package recently. Where do you think `UserContext` should live?